PR: https://github.com/odoo/odoo/pull/

From: c90348d4823079aa4586d24efa75e4457258ca44
From: Kinjal Mehta
Date: 2018-01-04 12:26:56

Structural Changes: 6
Total Changes: 6

[IMP] purchase: changed duplicate field label in the same model

================================= pseudo patch: =================================

--- a/addons/purchase/models/purchase.py
+++ b/addons/purchase/models/purchase.py
@@ -138,7 +138,7 @@ class PurchaseOrder(models.Model):
         ('invoiced', 'No Bill to Receive'),
         ], string='Billing Status', compute='_get_invoiced', store=True, readonly=True, copy=False, default='no')
 
-    picking_count = fields.Integer(compute='_compute_picking', string='Receptions', default=0, store=True)
+    picking_count = fields.Integer(compute='_compute_picking', string='Picking count', default=0, store=True)
     picking_ids = fields.Many2many('stock.picking', compute='_compute_picking', string='Receptions', copy=False, store=True)
 
     # There is no inverse function on purpose since the date may be different on each line

--- a/addons/purchase/models/res_config_settings.py
+++ b/addons/purchase/models/res_config_settings.py
@@ -12,7 +12,7 @@ class ResConfigSettings(models.TransientModel):
     po_order_approval = fields.Boolean("Order Approval", default=lambda self: self.env.user.company_id.po_double_validation == 'two_step')
     po_double_validation = fields.Selection(related='company_id.po_double_validation', string="Levels of Approvals *")
     po_double_validation_amount = fields.Monetary(related='company_id.po_double_validation_amount', string="Minimum Amount", currency_field='company_currency_id')
-    company_currency_id = fields.Many2one('res.currency', related='company_id.currency_id', readonly=True,
+    company_currency_id = fields.Many2one('res.currency', related='company_id.currency_id', string="Company Currency", readonly=True,
         help='Utility field to express amount currency')
     default_purchase_method = fields.Selection([
         ('purchase', 'Ordered quantities'),
@@ -21,7 +21,7 @@ class ResConfigSettings(models.TransientModel):
         help="This default value is applied to any new product created. "
         "This can be changed in the product detail form.", default="receive")
     module_purchase_requisition = fields.Boolean("Purchase Agreements")
-    group_warning_purchase = fields.Boolean("Warnings", implied_group='purchase.group_warning_purchase')
+    group_warning_purchase = fields.Boolean("Warnings for Purchase", implied_group='purchase.group_warning_purchase')
     module_stock_dropshipping = fields.Boolean("Dropshipping")
     group_manage_vendor_price = fields.Boolean("Vendor Pricelists",
         implied_group="purchase.group_manage_vendor_price")
